gravitee-apim-console-webui/src/management/api/portal/userGroupAccess/groups/api-portal-access-groups.component.scss

```diff
@@ -0,0 +1,37 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+@use '../../../../../scss/gio-layout' as gio-layout;
+
+:host {
+  @include gio-layout.gio-responsive-margin-container;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
```

Comment:
we have to find another solution we shouldn't have this here. we can make a 🤙 ?

Reply:
Save bar will be hidden by the list then a bit. To be corrected in the future...